### ISSUE TITLE
Manual Installation Documentation Issue

### ISSUE THREAD

**Issue by jonathan-klein-da** [smallstep/autocert#49]
### Subject of the issue
Following the manual steps for installing autocert into a kubernetes cluster we are asked to generate a /tmp/step.xxx directory. After generating the CA and Provisioner, creating the configmaps/secrets, and deploying the ca and autocert pods, I've run into an issue where the CA pod references /tmp/step.xxx directory insted of /home/step
### Environment
* Kubernetes version:
`Client Version: v1.21.5,  Server Version: v1.21.5-gke.1805`
* Cloud provider or hardware configuration:
`GKE`
* OS (e.g., from /etc/os-release):
`MACOS Monterey 12.1`
* Kernel (e.g., `uname -a`):
`Darwin Kernel Version 21.2.0: Sun Nov 28 20:28:54 PST 2021; root:xnu-8019.61.5~1/RELEASE_X86_64 x86_64 i386 MacBookPro16,1 Darwin`
* Install tools:
`Smallstep CLI/0.18.2 (darwin/amd64), Release Date: 2022-03-02 00:47 UTC`

### Steps to reproduce
- Follow the manual install steps at https://github.com/smallstep/autocert/blob/master/INSTALL.md#manual-install
- Select Standalone CA
- After steps are followed pods in CrashLoopBackOff
```
NAME                        READY   STATUS             RESTARTS   AGE
autocert-777f499f6f-h5x6r   0/1     CrashLoopBackOff   6          9m14s
ca-664ccd54d8-8tbrt         0/1     CrashLoopBackOff   5          4m11s
```
- Look at pod logs
`Error opening database of Type badgerv2 with source /tmp/step.fkH/db: error opening Badger database: Error Creating Dir: "/tmp/step.fkH/db": mkdir /tmp/step.fkH/db: no such file or directory`
- See that /tmp/step.xxx is refrenced
- Look at `config` configMap and see the refrence to `/tmp/step.xxx`
``` 
kubectl -n step get configmap config -o yaml

apiVersion: v1
data:
  ca.json: "{\n\t\"root\": \"/tmp/step.fkH/certs/root_ca.crt\",\n\t\"federatedRoots\":
    null,\n\t\"crt\": \"/tmp/step.fkH/certs/intermediate_ca.crt\",\n\t\"key\": \"/tmp/step.fkH/secrets/intermediate_ca_key\",\n\t\"address\":
    \":4443\",\n\t\"insecureAddress\": \"\",\n\t\"dnsNames\": [\n\t\t\"ca.step.svc.cluster.local\",\n\t\t\"127.0.0.1\"\n\t],\n\t\"logger\":
    {\n\t\t\"format\": \"text\"\n\t},\n\t\"db\": {\n\t\t\"type\": \"badgerv2\",\n\t\t\"dataSource\":
    \"/tmp/step.fkH/db\",\n\t\t\"badgerFileLoadingMode\": \"\"\n\t},\n\t\"authority\":
    {\n\t\t\"provisioners\": [\n\t\t\t{\n\t\t\t\t\"type\": \"JWK\",\n\t\t\t\t\"name\":
    \"admin\",\n\t\t\t\t\"key\": {\n\t\t\t\t\t\"use\": \"sig\",\n\t\t\t\t\t\"kty\":
    \"EC\",\n\t\t\t\t\t\"kid\": \"WKBOaYE72IvFzh1bOtRpeM6cl9aQDx5A8Rthrov7wDI\",\n\t\t\t\t\t\"crv\":
    \"P-256\",\n\t\t\t\t\t\"alg\": \"ES256\",\n\t\t\t\t\t\"x\": \"CJTqxPoBSpkMAlA3s1CWCag4eR8ERnmplLVzH4tV2io\",\n\t\t\t\t\t\"y\":
    \"hYGU4_n3oMdjVf_Qry6-WwYAP7A-MpL9IacocaBw5zE\"\n\t\t\t\t},\n\t\t\t\t\"encryptedKey\":
    \"eyJhbGciOiJQQkVTMi1IUzI1NitBMTI4S1ciLCJjdHkiOiJqd2sranNvbiIsImVuYyI6IkEyNTZHQ00iLCJwMmMiOjEwMDAwMCwicDJzIjoiYlVaNF9pQ3BzR1pjY3g3YUZ5RVJwZyJ9.R1xJvMScLN6dpIfi9HHlDtdM3MDk-VCXT3icZa5uH51_lVt_g7OFAw.Eg4Rs2if6EEUQJmq.oOL_4q_GS1HGh3o22AVWerSGmUL8AtxN9KQFYGXkgs32d_Xmsbl_GTpJL-UFCXY4tBiieK-1Lf4QEpsRv7uxBzWVfVzgTzlwuZKrER9yN8AoJqdHJ0mPCQtRvK-ZuiW489OcAz8aGslnwk3IdE61pq3XLQMkdsMvvf__XF07YDoQL9bMhfaUR5l2gxWSqjDXeBaX0Ms58Txf47kQRNYoKotAG8i1ffmwWVgHAtmuvr2Pu5rIlYNLeUooADFUg3G5IX8AFKgxkio_BMOoQGbKDtPxI-82toRMWa5ewPvEp9eAnkryWj9Sh__69IkQ0RhNFYLbQpjwkckFNNo0inI.vPNWIJFJDUdIhWbYxoasfA\"\n\t\t\t},\n\t\t\t{\n\t\t\t\t\"type\":
    \"JWK\",\n\t\t\t\t\"name\": \"autocert\",\n\t\t\t\t\"key\": {\n\t\t\t\t\t\"use\":
    \"sig\",\n\t\t\t\t\t\"kty\": \"EC\",\n\t\t\t\t\t\"kid\": \"ggp9W9moHEgPNgap2LFjQiTUbTQnwvp1IiV1RJG59ro\",\n\t\t\t\t\t\"crv\":
    \"P-256\",\n\t\t\t\t\t\"alg\": \"ES256\",\n\t\t\t\t\t\"x\": \"BRX4y2glUdC-DET-tCVrn9mJ3ZkJkPpN0sxcFddJXgE\",\n\t\t\t\t\t\"y\":
    \"_YCTwPekM8HweS3JJ9x7SNdK2VqbQRIRrr5daIYJJs0\"\n\t\t\t\t},\n\t\t\t\t\"encryptedKey\":
    \"eyJhbGciOiJQQkVTMi1IUzI1NitBMTI4S1ciLCJjdHkiOiJqd2sranNvbiIsImVuYyI6IkEyNTZHQ00iLCJwMmMiOjEwMDAwMCwicDJzIjoiNjFsNXJ3TWhobzJyT0pKbkJmM1VQUSJ9.jEa5eWzgAyfSKjAhIEMMPnlfVNQoU3hkJOd6fygv3J7sJZhNhAHwGg.R3MFjU-t-0sUnl6k.yvXl2HDfZgo9JKK6mvVMrytFc4RZmVnCzI_-1e3SKLrRgfDmMs3v4raVXYpZNfusucjmZMVUtMgGmmvWMzKeAsnhd3m_nyTju06gpOWaUmnJedwJRLxU8UhCa7iqgzDPdZUHgmk-PdypKBnglgJ7f2KrXD0sO1SPX_Vv0DIIm7I0ZpkTgPTMQxP-aKKKkp0s-wj50_JLZvOajQ-Vwa5RfFdkvqB428LpX1wT-HF0o3gCU2xq755mJeCzhkWLveDAWCp1qOnAa9p8nvBbfY-D5BlDJQpA45GfGKFKpIEazdGeWVD5omMkhac18_JAwbptb9r4HZkg-aZL5Qqbb28.vPezhzHn7UKBvDf5ycFliA\"\n\t\t\t}\n\t\t],\n\t\t\"template\":
    {},\n\t\t\"backdate\": \"1m0s\"\n\t},\n\t\"tls\": {\n\t\t\"cipherSuites\": [\n\t\t\t\"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256\",\n\t\t\t\"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256\"\n\t\t],\n\t\t\"minVersion\":
    1.2,\n\t\t\"maxVersion\": 1.3,\n\t\t\"renegotiation\": false\n\t}\n}\n"
  defaults.json: "{\n\t\"ca-url\": \"ca.step.svc.cluster.local\",\n\t\"ca-config\":
    \"/tmp/step.fkH/config/ca.json\",\n\t\"fingerprint\": \"897511bee3fe5b7419177fadfa6b791202b4c72829963aab756ca9bce660660b\",\n\t\"root\":
    \"/tmp/step.fkH/certs/root_ca.crt\"\n}"
kind: ConfigMap
metadata:
  creationTimestamp: "2022-04-06T15:17:58Z"
  name: config
  namespace: step
  resourceVersion: "52429"
  uid: feab43fe-06e0-4b48-b3c3-744acd680400
  ```

### Expected behaviour
Pods should come up with references to the correct mount points

### Actual behaviour
Pods come up with references to temp directory

### Additional context
Add any other context about the problem here

**Comment by jonathan-klein-da** [comment]
I was able to figure out the issue. I had to update configuration in `/tmp/step.xxx` to `/home/step`

This section in the documentation is problematic:
```
For older versions of step:

Run step ca init and follow prompts
Edit $(step path)/config/ca.json and change base paths to /home/step
Edit $(step path)/config/defaults.json to change base paths to /home/step and remove port from CA URL
$ sed -i "" "s|ca.step.svc.cluster.local:4443|ca.step.svc.cluster.local|" $(step path)/config/defaults.json
```
How do you know if you have an older version of step?

**Comment by maraino** [comment]
That documentation is 2 years old, I'm not sure what is referring to. The last released version is [v0.18.2](https://github.com/smallstep/cli/releases/tag/v0.18.2) (`step version`).

My recommendation to install autocert is to always use the [helm charts](https://github.com/smallstep/helm-charts/tree/master/autocert) with an existing deployment of `step-certificates`, that requires you to pass the `ca` block as defined in the previous link.

**Comment by dopey** [comment]
I linked that section of the documentation to our Github Releases page. Hopefully folks can check the latest release and see if they're far behind.
